### PR TITLE
Update Interop_OpenVPN.h

### PR DESCRIPTION
**Pull request by Hope4hope**
Safer connection

### CODE REVIEW

**Comment by moatazelmasry2** [comment]
Hi there. Thank you for your PR. Would you please consider using SHA256 or 5112 as default OPENVPN_DEFAULT_MD instead of RMD160?? since it is more standard. Thanks

File: src/Cedar/Interop_OpenVPN.h [hunk 1]
```diff
@@ -312,7 +312,7 @@ struct OPENVPN_SERVER_UDP
 };
 
 // OpenVPN Default Client Option String
-#define	OVPN_DEF_CLIENT_OPTION_STRING	"dev-type tun,link-mtu 1500,tun-mtu 1500,cipher AES-128-CBC,auth SHA1,keysize 128,key-method 2,tls-client"
+#define	OVPN_DEF_CLIENT_OPTION_STRING	"dev-type tun,link-mtu 1500,tun-mtu 1500,cipher AES-256-CBC,auth RMD160,keysize 128,key-method 2,tls-client"
```
Comment (moatazelmasry2) [inline src/Cedar/Interop_OpenVPN.h]: Why are AES_128 and SHA1 been removed? they are currently used in running setups, and this change will probably break them. Please add not replace. Unless the algorithm is really broken from a security POV

File: src/Cedar/Interop_OpenVPN.h [hunk 2]
```diff
@@ -139,8 +139,8 @@
 
 
 // The default algorithm
-#define	OPENVPN_DEFAULT_CIPHER					"AES-128-CBC"
-#define	OPENVPN_DEFAULT_MD						"SHA1"
+#define	OPENVPN_DEFAULT_CIPHER					"AES-256-CBC"
+#define	OPENVPN_DEFAULT_MD						"RMD160"
```
Comment (moatazelmasry2) [inline src/Cedar/Interop_OpenVPN.h]: Why RMD160 ??

**Comment by moatazelmasry2** [comment]
@Hope4hope ping!!!

**Comment by chipitsine** [comment]
@Hope4hope , are you here ?

**Comment by davidebeatrici** [comment]
Closing for lack of activity, feel free to reopen.

**Pull request closed without merging**